pkg/validation/errors/schema.go

```diff
@@ -166,6 +166,7 @@ func TooFewProperties(name, in string, n int64) *Validation {
 		Name:    name,
 		In:      in,
 		Value:   n,
+		Valid:   n,
```

Comment:
change the name of n to make it clear what it should be (minProperties), and make sure callers are setting it as expected